cmd/kube-spawn/init.go

```diff
@@ -79,10 +84,37 @@ func doInit() {
 
 	log.Println("Note: init on master can take a couple of minutes until every k8s pod came up.")
 
+	os.Remove(filepath.Join(kubeSpawnDir, "default/token"))
+
 	if err := nspawntool.InitializeMaster(k8srelease, nodes[0].Name); err != nil {
 		log.Fatalf("Error initializing master node %s: %s", nodes[0].Name, err)
 	}
 
+	token, err := getToken()
+	if err != nil {
+		log.Fatalf("Error reading token: %v", err)
+	}
+
+	if k8sruntime == "rkt" {
+		kubeadmContainerRuntime = "rktlet"
+	} else {
```

Comment:
Woudn't it be a little bit more correct if we will look for "docker" value explicitly? Like:

```
switch k8sruntime {
case "", "docker":
    kubeadmContainerRuntime = "docker"
case "rkt":
    kubeadmContainerRuntime = "rkt"
default:
    log.Fatalf("runtime %s is not supported", k8sruntime)
}
```